src/libs/decoders/opus.cpp

```diff
@@ -252,7 +267,7 @@ static int32_t opus_open(Sound_Sample * sample, const char * ext)
     int64_t pcm_result = op_pcm_total(of, -1); // If positive, holds total PCM samples
     internal->total_time = pcm_result == OP_EINVAL ? -1 : // total milliseconds in the stream
         static_cast<int32_t>
-		(ceil_divide(static_cast<uint64_t>(pcm_result), OPUS_SAMPLE_RATE_PER_MS));
+        (ceil_divide(static_cast<uint64_t>(pcm_result), OPUS_SAMPLE_RATE_PER_MS));
```

Comment:
cast-ception ;)

Perhaps `ceil_divide` should be a template function, so it could accept both unsigned and signed integers?

Reply:
Actually, just looking at that code, although I tried to mirror the SDL approach, I find it quite tangled. The fact it needs all the sign casting back-and-forth indicates a code smell that we're mixing logical values (unsigned sizes) with success codes (-1's vs positive).  Going to refactor it.

Reply:
Fixed in 3bbb5af3